app/femr/ui/views/history/indexEncounter.scala.html

```diff
@@ -51,6 +51,7 @@
                         <p><span class="infoLabel">First Name:</span> @outputStringOrNA(patient.getFirstName)</p>
                         <p><span class="infoLabel">Last Name:</span> @outputStringOrNA(patient.getLastName)</p>
                         <p><span class="infoLabel">Address:</span> @outputStringOrNA(patient.getAddress)</p>
+                        <p><span class="infoLabel">Phone Number:</span> @outputStringOrNA(patient.getPhoneNumber)</p>
```

Comment:
I had to break the formatting I've been using on this change with this phone number output.  The style I've been sticking to is: First Name, Last Name, Phone Number.  However, formatting of the page wouldn't allow this to display properly, so I pushed it down under Address.  If this is a problem, I can change it back.